### ISSUE TITLE
Apply security updates to EZID UI packages

### ISSUE THREAD

**Issue by jsjiang** [CDLUC3/ezid#519]
Apply security updates to the EZID UI packages

**Comment by jsjiang** [comment]
Related tickets #494

**Comment by jsjiang** [comment]
Joel provided fixes and documented changes in [Pull request for Update EZID UI build tool packages to fix critical vulnerabilities](https://github.com/CDLUC3/ezid/pull/520)

**Comment by jsjiang** [comment]
Deployed branch `ui-toolkit-updates` on ezid-dev for testing.

**Comment by jsjiang** [comment]
Functional test by running `verify_ezid_after_patching.py`: passed
UI tests: Maria and Jing performed UI  tests. All look good.

**Comment by jsjiang** [comment]
* Rushiraji performed UI test on ezid-dev
* Merged pull request
* created release tag v3.2.1 and release note
* deployed v3.2.1 on ezid-stg

**Comment by jsjiang** [comment]
@JoelCDL Hi Joel, I merged the pull request and the fixes cleared almost all Dependatbot alerts. Great job! Thank you!

There are still two more high security alerts (https://github.com/CDLUC3/ezid/security/dependabot):
* from the UI packages: "Prototype Pollution in lodash.merge"
* from EZID app: ReDos 

I will work on the EZID one. Can you take a look at the UI one and let me know if we need to do anything to clear it.

Thank you

Jing

**Comment by JoelCDL** [comment]
@jsjiang With the lodash.merge issue, you can click the "Create Dependabot security update" here, then merge in the change: https://github.com/CDLUC3/ezid/security/dependabot/8

**Comment by jsjiang** [comment]
@JoelCDL Got it. Thank you Joel!    -Jing

**Comment by jsjiang** [comment]
Failed to update `lodash.merge` due to conflicting dependencies:
```
Dependabot cannot update lodash.merge to a non-vulnerable version
The latest possible version that can be installed is 3.3.2 because of the following conflicting dependencies:

gulp-lb-include@0.3.1 requires lodash.merge@^3.1.0
gulp-lb-include@0.3.1 requires lodash.merge@^3.3.2 via lb-include@0.3.2
No patched version available for lodash.merge
The earliest fixed version is 4.6.2.
```

**Comment by jsjiang** [comment]
Deployed on ezid-prd 12/6.
